### PR TITLE
Create basic Azure Servicebus tests

### PR DESCRIPTION
**Pull request by ldrozdo**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests-jvm/azure-servicebus/pom.xml [hunk 1]
```diff
@@ -58,6 +58,22 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
```
Comment (jamesnetherton) [inline integration-tests-jvm/azure-servicebus/pom.xml:63]: Whenever you add new Camel Quarkus extension dependencies to the test, the corresponding `-deployment` dependency needs to be added to the `virtualDependencies` profile (you'll find it at the end of the `pom.xml`).

**Comment by ldrozdo** [comment]
@zbendhiba FYI :)

File: integration-tests-jvm/azure-servicebus/src/test/java/org/apache/camel/quarkus/component/azure/servicebus/it/AzureServiceBusTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public void basicProducerConsumerTest() {
+        final List sentMessages = RestAssured.given()
+                .get("/azure-servicebus/producer")
+                .then().statusCode(200).extract().body().as(List.class);
+
+        Assertions.assertEquals(inputBatch, sentMessages);
+    }
+
+    @Test
+    public void basicConsumerTest() {
```
Comment (aldettinger) [inline integration-tests-jvm/azure-servicebus/src/test/java/org/apache/camel/quarkus/component/azure/servicebus/it/AzureServiceBusTest.java:45]: Does the `basicConsumerTest` pass only when executed after the `basicProducerConsumerTest` ? If such ordering is needed, then we generally merge the two tests in a single one.

Comment (ldrozdo) [reply integration-tests-jvm/azure-servicebus/src/test/java/org/apache/camel/quarkus/component/azure/servicebus/it/AzureServiceBusTest.java]: @aldettinger you are right, I've fixed it.

**Pull request merged**